ansible/roles/rust/defaults/main.yml

```diff
@@ -1,7 +1,7 @@
 ---
 build_user: jenkins
 build_user_home_path: "/home/{{ build_user }}"
-rust_version: 1.29.2
+rust_version: stable
```

Comment:
Hi Stephen,

The issue with using 'stable' is they could change that at any time. Our products may not necessarily move at the same pace. It's generally better practice to 'pin' to a specific version. If we ran this Ansible code against a set of live slaves, and the stable version was moved on by Rust, we might end up with a version we can't build with.

Cheers,

Chris

Reply:
Ah fair point - I'll pick a specific version & change to that. Cheers